aspnetcore/mvc/controllers/areas.md

```diff
@@ -141,7 +141,7 @@ Set up a route definition that works with your newly created areas. The [🔧 Ro
    });
    ```
 
-Browsing to *http://<yourApp>/products*, the `Index` action method of the `HomeController` in the `Products` area will be invoked.
+Browsing to *http://\<yourApp\>/products*, the `Index` action method of the `HomeController` in the `Products` area will be invoked.
```

Comment:
make it `code` so it doesn't generate a clickable link - the MD is much easier to read as you don't escape it.

Thanks for fixing this.